README.md

```diff
@@ -228,6 +228,29 @@ After dealing with *CUDA* installing just install *Optix SKD*.
 
 - **[OptiX SDK](https://developer.nvidia.com/designworks/optix/download)** 
 
+### Android Build
+
+**Required:**
+
+- **[Android Studio](https://developer.android.com/studio)**
+- **[NDK r22b](https://developer.android.com/ndk/downloads#stable-downloads)**
+- **[Java 8](https://www.java.com/download/)**
```

Comment:
do you need the official Java or does Google recommend using its own or something?

Reply:
I couldn't find any recomendation, suppose official